lace/lace_stats/Cargo.toml

```diff
@@ -10,19 +10,19 @@ description = "Contains component model and hyperprior specifications"
 
 [dependencies]
 lace_utils = { path = "../lace_utils", version = "0.1.2" }
-lace_consts = { path = "../lace_consts", version = "0.1.2" }
-lace_data = { path = "../lace_data", version = "0.1.2" }
-special = "0.8"
+lace_consts = { path = "../lace_consts", version = "0.1.4" }
+lace_data = { path = "../lace_data", version = "0.1.3" }
```

Comment:
See here, you're bumping the `lace_data` version, so it seems like it should be bumped everywhere?